Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Extensions/FbDbContextOptionsExtensions.cs

```diff
@@ -0,0 +1,98 @@
+ *     you may not use this file except in compliance with the
+ *     License. You may obtain a copy of the License at
+ *     http://www.firebirdsql.org/index.php?op=doc&id=idpl
+ *
+ *     Software distributed under the License is distributed on
+ *     an "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either
+ *     express or implied.  See the License for the specific
+ *     language governing rights and limitations under the License.
+ *
+ *      Credits: Rafael Almeida (ralms@ralms.net)
+ *                              Sergipe-Brazil
+ *
+ *
+ *                              
+ *                  All Rights Reserved.
+ */
+
+
+using System;
+using System.Data.Common;
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Infrastructure.Internal;
+using Microsoft.EntityFrameworkCore.Utilities;
+using FirebirdSql.Data.FirebirdClient;
+
+
+namespace Microsoft.EntityFrameworkCore
+{
+    public static class FbDbContextOptionsExtensions
+    {
+        public static DbContextOptionsBuilder UseFirebird(
+            [NotNull] this DbContextOptionsBuilder optionsBuilder,
+            [NotNull] string connectionString,
+            [CanBeNull] Action<FbDbContextOptionsBuilder> FbOptionsAction = null)
+        {
+            Check.NotNull(optionsBuilder, nameof(optionsBuilder));
+            Check.NotEmpty(connectionString, nameof(connectionString)); 
+             
+            var extension = GetOrCreateExtension(optionsBuilder).WithConnectionString(connectionString);
+            ((IDbContextOptionsBuilderInfrastructure)optionsBuilder).AddOrUpdateExtension(extension);            
+            FbOptionsAction?.Invoke(new FbDbContextOptionsBuilder(optionsBuilder)); 
+            return optionsBuilder;
+        }
+
+        public static DbContextOptionsBuilder UseFirebird(
+            [NotNull] this DbContextOptionsBuilder optionsBuilder,
+            [NotNull] DbConnection connection,
+            [CanBeNull] Action<FbDbContextOptionsBuilder> FbOptionsAction = null)
+        {
+            Check.NotNull(optionsBuilder, nameof(optionsBuilder));
+            Check.NotNull(connection, nameof(connection)); 
+            var extension = GetOrCreateExtension(optionsBuilder).WithConnection(connection);
+            ((IDbContextOptionsBuilderInfrastructure)optionsBuilder).AddOrUpdateExtension(extension); 
+            FbOptionsAction?.Invoke(new FbDbContextOptionsBuilder(optionsBuilder)); 
+            return optionsBuilder;
+        }
+
+        public static DbContextOptionsBuilder<TContext> UseFirebird<TContext>(
+            [NotNull] this DbContextOptionsBuilder<TContext> optionsBuilder,
+            [NotNull] string connectionString,
+            [CanBeNull] Action<FbDbContextOptionsBuilder> FbOptionsAction = null)
+            where TContext : DbContext
+            => (DbContextOptionsBuilder<TContext>)UseFirebird(
```

Comment:
I would use full braces and body here.

Reply:
Do you mean as a general behavior ? Should i refactor each use of this shorten format usage ?

Reply:
Just here. I think the `=>` hurts the readability here. But it's just my opinion.

Reply:
If there is this simple feature why not use it?

Reply:
It does not affect the performance of the application.
And the code is simpler in that disrespect to return

Reply:
As I said. I don't think it improves readability.